lightning/src/util/message_signing.rs

```diff
@@ -12,13 +12,13 @@
 //! Note this is not part of the specs, but follows lnd's signing and verifying protocol, which can is defined as follows:
 //!
 //! signature = zbase32(SigRec(sha256d(("Lightning Signed Message:" + msg)))
-//! zbase32 from https://philzimmermann.com/docs/human-oriented-base-32-encoding.txt
+//! zbase32 from `<https://philzimmermann.com/docs/human-oriented-base-32-encoding.txt>`
```

Comment:
This doesn't create a hyperlink, it shows it as code: `<https://philzimmermann.com/docs/human-oriented-base-32-encoding.txt>`
```suggestion
//! zbase32 from <https://philzimmermann.com/docs/human-oriented-base-32-encoding.txt>
```

Reply:
Ah! Thanks. Is there a feature request to change the warning message to drop the "`"s on rustdoc, or should I open one?

Reply:
There's no issue, you should open a new one.

That seems be hard to do, this is part of diagnostic machinery. How would you decide what suggestions get a backtick and which don't?

Reply:
Yea, I went and read some of the rustc source after commenting and realized its probably not trivial. I'll leave it alone for now, I was hoping it was pretty trivial and I could open a feature request someone could knock out quick, but I don't want to go demand something thats a papercut and takes significant effort to fix.